hello_world.cpp

```diff
@@ -1,7 +1,6 @@
 #include "sycl_ext_complex.hpp"
 #include <cassert>
 #include <complex>
-#include <sycl/sycl.hpp>
```

Comment:
Do you mind putting it back? :) I know it's not portable, but I find it ugly to depend on some transient header for a hello-world

Reply:
I don't care about clean history :D I will squatch everything in one big commit. This is also I prefer smaller PR

Reply:
I generally interactive rebase and try to keep each commit working and single purpose, in case anything needed to reverted, so I don't think this will need to be squashed.